test/index.spec.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @ridedott/no-single-line-comment-block */
```

Comment:
this can be overridden in `.eslintrc` for `*.spec.ts` files only :D

Reply:
if it's in .eslintrc test are failing as it's expected to fail in one of the new tests

Reply:
I think this should work
```
  - files:
      - test/**.spec.ts
    rules:
      @ridedott/no-single-line-comment-block: off
```